sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/sweep.py

```diff
@@ -300,6 +306,7 @@ def _to_job(self) -> SweepJob:
             outputs=self._job_outputs,
             identity=self.identity,
             queue_settings=self.queue_settings,
```

Comment:
also update from_rest_object? code can be seen [here](https://github.com/Azure/azure-sdk-for-python/pull/30746/files#diff-0f42448c92420aed5001ee117eb7d4c482ac8b24f5ab92e935baad41761ea2aeR285-R287):
```python
if "resources" in obj:
    obj["resources"] = JobResourceConfiguration(**obj["resources"])
```

Reply:
Discussed offline, we are using set_resources, similar like set_limits